### PR TITLE
Reload Zeus Interface Keybind

### PR DESCRIPTION
**Pull request by neilzar**
**When merged this pull request will:**
- Add a keybind to reload the Zeus interface when needed.
- Hack for #152 but not a fix

### CODE REVIEW

File: addons/editor/stringtable.xml [hunk 1]
```diff
@@ -222,5 +222,11 @@
             <Japanese>全編集可アイコンの視認性を切り替えます。一時的な処理軽減や整理に使用できます。</Japanese>
             <Polish>Przełącza widoczność wszsytkich ustawionych ikon, przydatne by tymczasowo zmniejszyć lag, bądź niepotrzebne zaśmiecenie ekranu gdy znajduje się na nim dużo ikon.</Polish>
         </Key>
+        <Key ID="STR_ZEN_Editor_ReloadDisplay">
+            <English>Reload Display</English>
+        </Key>
+        <Key ID="STR_ZEN_Editor_ReloadDisplay_Description">
+            <English>Reloads the Display to fix the UI lockup bug.</English>
```
Comment (CreepPork) [inline addons/editor/stringtable.xml:229]:
```suggestion
            <English>Reloads the Zeus interface (can be used to fix various lockup issues).</English>
```

File: addons/editor/initKeybinds.sqf [hunk 2]
```diff
@@ -100,3 +100,14 @@
         };
     };
 }, {}, [0, [false, false, false]]] call CBA_fnc_addKeybind; // Default: Unbound
+
+// Hack fix for the Zeus lockup bug, will reopen Zeus interface.
+[ELSTRING(main,DisplayName), QGVAR(reloadDisplay), [LSTRING(ReloadDisplay), LSTRING(ReloadDisplay_Description)], {
+    if (!isNull curatorCamera && {!GETMVAR(RscDisplayCurator_search,false)}) then {
+        findDisplay 312 closeDisplay 2;
```
Comment (CreepPork) [inline addons/editor/initKeybinds.sqf:107]: No magic numbers.

File: addons/editor/initKeybinds.sqf [hunk 3]
```diff
@@ -100,3 +100,14 @@
...
+
+        {openCuratorInterface} call CBA_fnc_execNextFrame;
```
Comment (CreepPork) [inline addons/editor/initKeybinds.sqf:109]:
```suggestion
        [{ openCuratorInterface }] call CBA_fnc_execNextFrame;
```

**Review by CreepPork — CHANGES_REQUESTED** [review]

File: addons/editor/initKeybinds.sqf (hunk 3 above)
Comment (neilzar) [reply addons/editor/initKeybinds.sqf:109]: There is already another spot where it is done like this and works: https://github.com/zen-mod/ZEN/blob/e2a7f8b695a4d19a4880dc1db6be9a10b43f294c/addons/remote_control/functions/fnc_start.sqf#L79

Comment (CreepPork) [reply addons/editor/initKeybinds.sqf:109]: I think that @mharis001 has a way of always putting things in `[]` even if there is only one argument.

Comment (neilzar) [reply addons/editor/initKeybinds.sqf:109]: Well, that is his code according to blame.

**Review by CreepPork — APPROVED** [review]

**Comment by Kexanone** [comment]
I guess we can close this one, since we have now a proper fix.

**Review by Kexanone — APPROVED** [review]

**Pull request merged**